subgraph/src/PolicyRegistry.ts

```diff
@@ -1,5 +1,6 @@
 import { PolicyUpdate } from "../generated/PolicyRegistry/PolicyRegistry";
 import { Court } from "../generated/schema";
+import { ipfs, json, Bytes, TypedMap, JSONValue } from "@graphprotocol/graph-ts";
```

Comment:
I would prefer if we could do without the `ipfs` query as it's not clear if it will be supported by the decentralized network.

Reply:
removed